v3/src/components/graph/components/inspector-panel/measure-panel.scss

```diff
@@ -0,0 +1,22 @@
+.measure-group-button {
+  justify-content: left!important;
```

Comment:
Are all the instances of `!important` in this file necessary? Or are they just remnants of trying to troubleshoot the styling issue that has since been fixed?